src/test/java/io/aiven/kafka/connect/common/output/JsonOutputWriterTestHelper.java

```diff
@@ -66,6 +66,7 @@ protected void assertRecords(final List<SinkRecord> records, final String expect
         for (int i = 0; i < records.size(); i++) {
             sut.writeRecord(records.get(i));
         }
+        sut.close();
```

Comment:
If we split writing and checking, we could use try-with-resource: 
```java
try (final OutputWriter sut = <build>) {
    // do the writing
} // close is called automatically here
// do the checks
```

Reply:
But in this case I have to get rid of assertRecord helping function:
System under test object should be created in a test body function , otherwise, we fully hide an essence of test from a test reader including instance building. If SUT created in each test body writing also have to be there because of try function. So there is only assert will be left in a helper, and this is not very helping. So, for me it looks like using `try` function make testing more verbose because code that was moved to `assertRecord` helping function now should be copied to a body of every function, that use it. Not sure if this is something we want to achieve here.

 On the other hand, if you have any idea how to make tests smaller and use a  try-with-resource, I'm all ears.